ios/app/src/AppDelegate.m

```diff
@@ -35,6 +35,7 @@ -             (BOOL)application:(UIApplication *)application
 
     jitsiMeet.defaultConferenceOptions = [JitsiMeetConferenceOptions fromBuilder:^(JitsiMeetConferenceOptionsBuilder *builder) {
         [builder setFeatureFlag:@"resolution" withValue:@(360)];
+        [builder setFeatureFlag:@"ios.screensharing.extension" withValue:@"org.jitsi.meet.broadcast.extension"];
```

Comment:
Let's have this in the plist as well please so the WebRTC part needs no parameters.

Reply:
moved to WebRTC